### ISSUE TITLE
Fetch the actual schemaId in the indy vdr and indy sdk anoncreds registries

### ISSUE THREAD

**Issue by TimoGlastra** [hyperledger/aries-framework-javascript#1259]
`AnonCredsRegistry.getCredentialDefinition()` should return the actual 
`schemaId` for both the IndySdkAnonCredsRegistry and the IndyVdrAnonCredsRegistry. However, currently it will use the `ref`, which is actually the transaction seqNo and not the string schema identifier.

To fix this, we need to resolve the schema associated with the seqNo. Both the indy sdk and indy vdr support resolving a schema using the seqNo in addition to the schemaId.

Create a private method in both registry implementations called `fetchIndySchema`. This should take the agent context and a schemaId or ref as input.  It should then call the ledger (differs for indy vdr and indy-sdk) and return the schema. The `getSchema` method should be updated to call this `fetchIndySchema`, and we should also call this method in the `getCredentialDefinition` method of both registries. As the `ref` or `schemaId` returned by the indy-sdk / indy-vdr is actually the sequence number.

**Comment by Vickysomtee** [comment]
@TimoGlastra the schemaId is a string but in the response returned from the Ledger, there is seqNo. That should be the actual sequence number?

**Comment by Vickysomtee** [comment]
Also, it doesn't seem like we need the `fetchIndySchema` in the ` getCredentialDefinition` method

**Comment by TimoGlastra** [comment]
indyVdr -> getCredentialDefinition:
```
{
  result: {
   ref: 123123 // or '12313123'
  }
}
```


return:

```
{
  schemaId: 'sdfjaksdfj:2:Example Schema:1.0.0'
}
```


what is returned now:

```
{
  schemaId: '12313123' // this is the seqNo, not hte schemaId
}
```

What we have to do is getSchem (by the seqNo == transactionNo == ref), and the replace the seqNo with the actual string schema id

**Comment by TimoGlastra** [comment]
Fixed by #1290
